index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio</title>
+  </head>
+
+  <body>
+    <header class="container">
+      <a href="#"><p id="logo-placeholder">Welcome</p></a>
+
+      <nav>
+        <img src="images/burger-menu.png" alt="menu icon" id="burger-menu" />
+      </nav>
+    </header>
+
+    <section id="headline" class="container">
+      <h1>
+        Hey there. I’m Hayet <br />
+        <span class="color-green">I’m a software developer</span>
+      </h1>
+      <p>
+        I can help you build a product, feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hesitiate to contact me.
+      </p>
+
+      <ul id="social-media" class="container">
+        <li><a href="https://twitter.com/hayet_fe"> <img src="images/twitter.png" alt="twiiter icon" class="icon" /></a></li>
+        <li><a href="https://www.linkedin.com/in/hayet-f-5b4347247/"><img src="images/linkedin.png" alt="linkedin icon" class="icon" /></a></li>
+        <li><a href="#"> <img src="images/medium.png" alt="medium icon" class="icon" /></a></li>
+        <li><a href="https://github.com/laiifuu"><img src="images/github.png" alt="github icon" class="icon" /></a></li>
+        <li><a href="#"><img src="images/angellist.png" alt="angellist icon" class="icon" /></a></li>
+      </ul>
+    </section>
```

Comment:
- [ ] [OPTIONAL] your project looks great and so clean I'm impressed 💯 👏 , kindly add this button as a link since it do belong to this section 


![btttttttttttttttn](https://user-images.githubusercontent.com/89282221/185396571-802b84fa-1a19-4225-9be3-82c2f919acce.png)